sbt-scalajs-cross/src/main/scala/scalajscross/JSPlatform.scala

```diff
@@ -0,0 +1,16 @@
+package scalajscross
+
+import sbtcross._
+
+import sbt._
+
+import org.scalajs.sbtplugin.ScalaJSCrossVersion
+
+case object JSPlatform extends Platform {
+  def identifier: String = "js"
+  def sbtSuffix: String  = "JS"
+  def enable(project: Project): Project =
+    project.enablePlugins(ScalaJSCrossPlugin)
```

Comment:
This should enable `ScalaJSPlugin`, not `ScalaJSCrossPlugin`. `ScalaJSCrossPlugin` should be automatically triggered by `ScalaJSPlugin`, as per my comment in `ScalaJSCrossPlugin`.